runtime/kusama/src/lib.rs

```diff
@@ -1388,7 +1386,6 @@ construct_runtime! {
 	{
 		// Basic stuff; balances is uncallable initially.
 		System: frame_system::{Pallet, Call, Storage, Config, Event<T>} = 0,
-		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 32,
```

Comment:
should it be replaced with a dummy that occupies the index or is it fine to overwrite it (in which case what's the point of the index?)?

Reply:
Nothing needs to occupy the index. The index is used to generate a unique identifier for the metadata.

Reply:
To test my understanding:
1. Index is used to ensure stable identifier for the metadata when a new pallet is introduced (and not depend on the order of declaration).
2. It's fine to reuse an index later as long there is migration in place?

Reply:
Yeah, assuming you clean up the old storage items, and that you update your tx version